python/lsst/ctrl/execute/slurmPlugin.py

```diff
@@ -119,11 +119,16 @@ def submit(self, platform, platformPkgDir):
     def loadSlurm(self, name, platformPkgDir):
         if self.opts.reservation is not None:
             self.defaults["RESERVATION"] = (
-                "#SBATCH --reservation=%s" % self.opts.reservation
+                "#SBATCH --reservation %s" % self.opts.reservation
             )
         else:
             self.defaults["RESERVATION"] = ""
 
+        if self.opts.qos is not None:
```

Comment:
Since self.opts.qos is a str --> if self.opts.qos:
(https://developer.lsst.io/python/style.html#is-and-is-not-should-be-used-when-comparing-to-none)